docs/tutorials/notes-app/includes/navigation.md

```diff
@@ -0,0 +1,336 @@
+
+Currently the **note** view displays a single note, and there isn't a view that represents multiple notes. To display multiple notes, create a new view and model: **AllNotes**.
+
+01. In the **Solution Explorer** pane, right-click on the **Views** folder and select **Add** > **New Item...**
+01. In the **Add New Item** dialog, select **.NET MAUI** in the template list on the left-side of the window. Next, select the **.NET MAUI ContentPage (XAML)** template. Name the file _AllNotesPage.xaml_, and then select **Add**.
+01. In the **Solution Explorer** pane, right-click on the **Models** folder and select **Add** > **Class...**
+01. Name the class **AllNotes.cs** and press **Add**.
+
+### Code the AllNotes model
+
+The new model will represent the data required to display multiple notes. This data would be a property that represents a collection of notes. The collection will be an `ObservableCollection` which is a specialized collection. When a control lists multiple items, such as a `ListView`, is bound to an `ObservableCollection`, the two work together to automatically keep the list of items in sync with the collection. If the list adds an item, the collection is updated. If the collection adds an item, the control is automatically updated with a new item.
+
+01. In the **Solution Explorer** pane, open the _Models\\AllNotes.cs_ file.
+01. Replace the code with the following snippet:
+
+    :::code language="csharp" source="../snippets/navigation/csharp/Notes/Models/AllNotes.cs":::
+
+The previous code declares a collection, named `Notes`, and uses the `LoadNotes` method to load notes from the device. This method uses Linq extensions to load, transform, and sort the data into the `Notes` collection.
+
+### Design the AllNotes page
+
+Next, the view needs to be designed to support the **AllNotes** model.
+
+01. In the **Solution Explorer** pane, open the _Views\\AllNotesPage.xaml_ file.
+01. Replace the code with the following markup:
+
+    :::code language="xaml" source="../snippets/navigation/csharp/Notes/Views/AllNotesPage.xaml":::
+
+The previous XAML introduces a few new concepts:
+
+- The `ContentPage.ToolbarItems` property contains a `ToolbarItem`. The buttons defined here usually display at the top of the app, along the page title. Depending on the device, though, it may be in a different position. When one of these buttons is pressed, the `Clicked` event is raised, just like a normal button.
+
+  The `ToolbarItem.IconImageSource` property sets the icon to display on the button. The icon can be any image resource defined by the project, however, in this example, a `FontImage` is used. A `FontImage` can use a single glyph from a font as an image.
+
+- The `CollectionView` control displays a collection of items, and in this case, is bound to the model's `Notes` property. The way each item is presented by the collection view is set through the  `CollectionView.ItemsLayout` and `CollectionView.ItemTemplate` properties.
```

Comment:
I really hate pounding sentences with `code` because I find it visually distracting. So sometimes I try to explain thigs in normal terms, such as "collection view" instead of `CollectionView`. How do you feel about that?

Reply:
I'm normally a stickler for using `code` but at the same time I've hit the same issue as you while writing the handler docs. I ended up with impenetrable sentences full of `code`. So I've ended up taking your approach now. Hence you can ignore this.